simtools/simulator.py

```diff
@@ -333,7 +333,7 @@ def _collectArrayModelParameters(self, configData):
             raise
 
         # Reading telescope keys
-        telKeys = [k for k in _restData.keys() if k[0:2] in ["L-", "M-", "S-"]]
+        telKeys = [k for k in _restData.keys() if k[1:4] in ["ST-", "SC-"]]
```

Comment:
I don't understand this line. This would pick up all telescopes except SCTs. For SCT this should be `CT-`, no?